addons/account/static/src/components/account_merge_wizard_line_one2many/account_merge_wizard_line_one2many.js

```diff
@@ -0,0 +1,68 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import {
+    SectionAndNoteListRenderer,
+    SectionAndNoteFieldOne2Many,
+    sectionAndNoteFieldOne2Many,
+} from "../section_and_note_fields_backend/section_and_note_fields_backend";
+import { onWillStart } from "@odoo/owl";
+
+export class AccountMergeWizardLinesRenderer extends SectionAndNoteListRenderer {
+    setup() {
+        super.setup();
+        this.titleField = "info";
+    }
+
+    getCellClass(column, record) {
+        const classNames = super.getCellClass(column, record);
+        // Even though the `is_selected` field is hidden for section lines, we should
+        // keep its column to align the section header name with the account names.
```

Comment:
But `invisible` in a list view doesn't remove the column :thinking: (`column_invisible` does)

Reply:
(so: the reason why we do this, is that the inherited method removes all columns except `titleField`. We need to override this to bring back `is_selected`'s column so that it still exists in section lines)